app/views/config_reports/show.html.erb

```diff
@@ -20,20 +19,6 @@
   link_to(_("Host details"), @config_report.host, :class => 'btn btn-default'),
   link_to(_("Other reports for this host"), host_config_reports_path(@config_report.host), :class => 'btn btn-default')
 %>
+  <div id="diff-modal-editor" class="modal-body"></div>
```

Comment:
nitpick: extrawhitespaces before the the diff, should be with no leading spaces.